app/controllers/api/v1/game_plays_controller.rb

```diff
@@ -1,4 +1,4 @@
-class Api::V1::GamePlayApisController < ApplicationController
+class Api::V1::GamePlaysController < ApplicationController
```

Comment:
great minds think alike! Should it not be single though... GamePlayController?

Reply:
we agree that gamepleay will be plural and endgame singular